src/viewer.js

```diff
@@ -2132,6 +2132,58 @@ $.extend( $.Viewer.prototype, $.EventSource.prototype, $.ControlDock.prototype,
      */
     _cancelPendingImages: function() {
         this._loadQueue = [];
+    },
+
+    /**
+     * Remove the reference strip.
+     * @function
+     * @return {OpenSeadragon.Viewer} Chainable.
+     */
+    removeReferenceStrip: function() {
+        this.showReferenceStrip = false;
+
+        if (this.sequenceMode) {
+            if (this.referenceStrip) {
+                this.referenceStrip.destroy();
+                this.referenceStrip = null;
+            }
+        }
+
+        return this;
+    },
+
+    /**
+     * Display the reference strip based on the currently set tileSources.
+     * @function
+     * @return {OpenSeadragon.Viewer} Chainable.
+     */
+    addReferenceStrip: function() {
+        this.showReferenceStrip = true;
+
+        if (this.sequenceMode) {
+            if (this.referenceStrip) {
+                this.referenceStrip.destroy();
+                this.referenceStrip = null;
+            }
+
+            if (this.tileSources.length && this.tileSources.length > 1) {
```

Comment:
Well, reference strip won't be created in Viewer.open() if there is only 1 tile source (i.e. `$.isArray(tileSources)` needs to be fulfilled)).

This check (`this.tileSources.length > 1`) follows that rule, but can't use the same syntax as now tile sources can no longer be checked for array type because array is "force-created" later in the open() function:
```
        if (!$.isArray(tileSources)) {
            tileSources = [tileSources];
        }
```
What the original (`$.isArray(tileSources)` ) check implies makes sense to me -- not to show reference strip for single image source (even if sequence mode is on).

So, I'd leave it for consistency of implementation. Otherwise, the corresponding check  in the Viewer.open() also needs to be removed, IMO.